services/movie_session.py

```diff
@@ -42,3 +44,16 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> List[Dict[str, int]]:
+    tickets = (
+        Ticket.objects
+        .filter(movie_session_id=movie_session_id)
+        .values("row", "seat")
+    )
+    taken_seats = [
+        {"row": ticket["row"], "seat": ticket["seat"]}
+        for ticket in tickets
+    ]
```

Comment:
why do u need this list comprehension if u are using values method ?

Reply:
Because the task was to return list of dictionaries, while values method will return QuerySet of dictionaries.

Reply:
u can just make list(QuerySet) to get list instead of queryset, no need to make loop for this

Reply:
Idk why I was sure, that, even though QS returns dicts with k-v, since we using values, it will return objects if I use list on QS) Thanks for clarifying.